### ISSUE TITLE
WriteTo.ColoredConsole() ---- Not available

### ISSUE THREAD

**Issue by DanJ210** [serilog/serilog#963]
I've followed the procedure exactly as described but when I try to use ColoredConsole, it does not show up. I'm not sure what I'm doing wrong or missing. I am new at this so it could be me but thought I'd let everyone know just in case.

**Comment by khellang** [comment]
> I've followed the procedure exactly as described

It would be of great help if you would let ut know that procedure is :smile:

- Did you install the `Serilog.Sinks.ColoredConsole` NuGet package successfully?
- What was the output in the [Package Manager Console](https://docs.microsoft.com/en-us/nuget/tools/package-manager-console) after installing it?
- Have you checked that the project actually references the assembly from the package?

Given the information you've provided, I'm not sure what we can do 😛

**Comment by DanJ210** [comment]
Oh sure no problem. It's the getting started that I followed from the main serilog website which took me here and I just assumed that was the only page. My apologies. 

Everything in the PMC said it finished no problem and the application builds. I did follow the rest of the getting started. I am using the logger to log to the console window but I don't have colored console. Upon double checking the link I've showed bellow. I don't see anywhere that it says to install the ColoredConsole package but does use .ColoredConsole in configuration as if that's included in serilog core. Am I missing something from that? I am new to this.

I did install RollingFile though but I cannot use that as it turns everything red when I do. I am using .Net Core Web App, not console so I figured that was why. Check the link below and see where it says to install ColoredConsole, if not then maybe this helps? Otherwise I can search and install that package and then try it.

https://github.com/serilog/serilog/wiki/Getting-Started

**Comment by khellang** [comment]
> I don't see anywhere that it says to install the ColoredConsole package but does use .ColoredConsole in configuration as if that's included in serilog core. Am I missing something from that? I am new to this.

Ah, yeah, that's probably something left from the 1.x days when the console sinks was in the core package. Nowadays, all sinks are in their own packages. Try installing `Serilog.Sinks.ColoredConsole `, (I actually prefer `Serilog.Sinks.Literate` myself) and see if that helps.

> I did install RollingFile though but I cannot use that as it turns everything red when I do.

It turns everything red? The console? Visual Studio?

> I am using .Net Core Web App, not console so I figured that was why.

The rolling file sink should work well with a .NET Core web app.

**Comment by nblumhardt** [comment]
I've added `Install-Package Serilog.Sinks.Literate` and changed `ColoredConsole()` to `LiterateConsole()` in the example, thanks @DanJ210.

If you are still having trouble getting the rolling file sink installed, can you let us know more details? As Kristian suggested, it _should_ work fine with .NET Core, but the tooling around it doesn't always line up :-)

**Comment by DanJ210** [comment]
I'm still playing with everything to see how it all works. I may prefer literate then. I'm sure installing ColoredConsole package does work and if it doesn't I will post back here.

> It turns everything red? The console? Visual Studio?

I meant everything inside the code block for -

`public Startup(IHostingEnvironment env) {
            Log.Logger = new LoggerConfiguration()
            //.MinimumLevel.Debug()
            //.Enrich.FromLogContext()
            .WriteTo.LiterateConsole()
            .WriteTo.RollingFile("logs\\myapp-{Date}.txt")
            .CreateLogger();
        }`

But I realized I had an extra semi colon so no longer does it turn anything red. That was my fault. I'm actually seeing the file that I created and the logs that I'm writing to the file just to test the different kinds!

> The rolling file sink should work well with a .NET Core web app.

It does seem to work and so far it works well. Thanks for helping me with not just the original issue but also helping me with writing proper helpful issues and syntax for posts here on GitHub! Lol.
